<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testQT5Class</class>
 <widget class="QMainWindow" name="testQT5Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testQT5</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开数据库</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_addSQLite"/>
   <addaction name="action_pan"/>
   <addaction name="action_selectEnv"/>
   <addaction name="action_update"/>
   <addaction name="action_searchEnv"/>
   <addaction name="action_clearalllayer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_pan">
   <property name="text">
    <string>漫游</string>
   </property>
  </action>
  <action name="action_selectEnv">
   <property name="text">
    <string>矩形选择</string>
   </property>
   <property name="toolTip">
    <string>矩形选择</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="action_searchEnv">
   <property name="text">
    <string>矩形查询</string>
   </property>
  </action>
  <action name="action_addSQLite">
   <property name="text">
    <string>添加本地路径</string>
   </property>
  </action>
  <action name="action_clearalllayer">
   <property name="text">
    <string>清除所有图层</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testqt5.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>helloworld()</slot>
  <slot>setLabelString()</slot>
 </slots>
</ui>
